base/initdefs.jl

```diff
@@ -353,6 +353,7 @@ end
 const atexit_hooks = Callable[
     () -> Filesystem.temp_cleanup_purge(force=true)
 ]
+const _atexit_lock = ReentrantLock()
```

Comment:
```suggestion
const _atexit_hooks_lock = ReentrantLock()
```

Reply:
Thanks, yeah that seems like a nice change 👍